swagger/tekton-dashboard-api.yaml

```diff
@@ -0,0 +1,788 @@
+openapi: 3.0.0
+info:
+  title: Tekton Dashboard
+  description: >-
+    Tekton Dashboard API provides the REST API access to the Tekton Pipeline resources.
+  version: 0.0.1
+servers:
+  - url: '{protocol}://{host}:{port}/v1/namespaces/{namespace}'
+    description: 'Running in the tekton-dashboard pod'
+    variables:
+      protocol:
+        enum:
+          - http
+          - https
+        default: http
+      host:
+        default: localhost
+      port:
+        default: '9097'
+      namespace:
+        default: tekton-pipelines
+        description: >-
+          namespace where the Tekton resources (Pipelines, Tasks, etc.) are
+          installed
+paths:
+  /pipeline/:
```

Comment:
https://github.com/tektoncd/dashboard/pull/155 updated the APIs to use plurals, so for example here `pipeline` should be `pipelines`

Reply:
I updated them.